### PR TITLE
[mqtt.homeassistant] Add support for Scene component

### PR DESCRIPTION
**Pull request by ccutrer**

### CODE REVIEW

**Review by lolodomo — APPROVED** [review]
LGTM

**Comment by lolodomo** [comment]
Waiting for @antroids approval before merging because my knowledge of this binding is too much limited.

**Review by antroids — APPROVED** [review]
LGTM

**Pull request merged**
